src/Core/Dfe.Complete.Application/Projects/Queries/GetProject/GetProjectGroupDetailsById.cs

```diff
@@ -0,0 +1,72 @@
+using Dfe.Complete.Domain.Interfaces.Repositories;
+using Dfe.Complete.Application.Common.Models;
+using Dfe.Complete.Application.Projects.Models;
+using Dfe.Complete.Domain.Entities;
+using Microsoft.Extensions.Logging;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using Dfe.AcademiesApi.Client.Contracts;
+using Dfe.Complete.Domain.Enums;
+using Dfe.Complete.Domain.ValueObjects;
+using Dfe.Complete.Utils;
+using Exception = System.Exception;
+
+namespace Dfe.Complete.Application.Projects.Queries.GetProject
+{
+    public record GetProjectGroupDetailsQuery(ProjectGroupId ProjectGroupId) : IRequest<Result<ProjectGroupDetails>>;
+
+    public class GetProjectGroupDetailsQueryHandler(ICompleteRepository<ProjectGroup> projectGroupRepository,
```

Comment:
```suggestion
    public class GetProjectGroupDetailsQueryHandler(IProjectGroupReadRepository projectReadRepository,
``` Create IProjectGroupReadRepository like IProjectReadRepository is created and use it as going forward, we are no longer using ICompleteRespository for ready operations